### PR TITLE
chore(build): add needed includes for linking with tensorrt dd

### PR DESCRIPTION
**Pull request by Bycob**

### CODE REVIEW

File: src/backends/torch/torchlib.cc [hunk 1]
```diff
@@ -1550,6 +1550,22 @@ namespace dd
         unsupo.finalize(output_params, out,
                         static_cast<MLModel *>(&this->_mlmodel));
       }
+
```
Comment (beniz) [inline src/backends/torch/torchlib.cc:1553]: I believe this belongs to another PR !

**Review by fantes — APPROVED** [review]

File: CMakeLists.txt [hunk 2]
```diff
@@ -813,6 +813,7 @@ if (USE_TORCH)
       "-DCMAKE_CXX_FLAGS=-isystem ${CMAKE_BINARY_DIR}/pytorch/src/pytorch/ -isystem ${SPDLOG_INCLUDE_DIR} -isystem ${PROTOBUF_INCLUDE_DIR}"
       "-DCMAKE_CUDA_FLAGS=-isystem ${CMAKE_BINARY_DIR}/pytorch/src/pytorch/ -isystem ${SPDLOG_INCLUDE_DIR} -isystem ${PROTOBUF_INCLUDE_DIR}"
       -DCMAKE_CUDA_COMPILER=/usr/local/cuda/bin/nvcc
+    BUILD_COMMAND ""
```
Comment (Bycob) [inline CMakeLists.txt:816]: This was accidentally commited, but it prevents pytorch_vision to rebuild twice, so we can let it there? (the library is built at install step anyway)

Comment (fantes) [reply CMakeLists.txt:816]: ok for me

**Review by beniz — APPROVED** [review]
